lldb/source/Plugins/Platform/WebAssembly/PlatformWasmProperties.td

```diff
@@ -0,0 +1,19 @@
+include "../../../../include/lldb/Core/PropertiesBase.td"
+
+let Definition = "platformwasm" in {
+  def RuntimePath : Property<"runtime-path", "FileSpec">,
+                    Global,
+                    DefaultStringValue<"">,
+                    Desc<"Path to the WebAssembly runtime binary.">;
+  def RuntimeArgs : Property<"runtime-args", "Args">,
+                    Global,
+                    DefaultStringValue<"">,
+                    Desc<"Extra arguments to pass to the WebAssembly runtime.">;
+  def PortArg : Property<"port-arg", "String">,
+                Global,
+                DefaultStringValue<"">,
+                Desc<"Argument to the WebAssembly runtime to specify the "
+                     "GDB remote port. The port number chosen by LLDB will be "
+                     "concatenated to this argument. For example: "
+                     "-g=127.0.0.1: or --debugger-port.">;
```

Comment:
This is ok but the pedant in me wants to know whether this is an example of what I should enter, or what I would expect lldb to produce from what I enter.

Also, `--debugger-port` presumably would need to be `--debugger-port ` with a space, so it's probably a worse example to put here.

Edit: worse as in harder to explain in few words, I see that it's useful though so keep if if you want.

How about:
For example setting this to "-g=127.0.0.1:" will result in "-g127.0.0.1:<port number>".

Reply:
I suggested only keeping the `-g` example but now I realise because of the space issue, `--debugger port ` is good to include.